core-request.html

```diff
@@ -64,7 +71,16 @@
        */
       completes: {
         type: Promise,
-        readOnly: true
+        readOnly: true,
+        notify: true,
+        value: function() {
+          var completes = new Promise(function (resolve, reject) {
```

Comment:
I've addressed this by integrating polymerlabs/promise-polyfill for now. Let's explore this issue further down the road and decide if Polymer elements should be Promise-free at that time.